examples/nft/assets/approval_program.teal

```diff
@@ -0,0 +1,103 @@
+#pragma version 2
```

Comment:
Please add more description of this smart-contract.

Reply:
ok